core-plugins/src/main/java/co/cask/hydrator/plugin/batch/aggregator/RowDenormalizerConfig.java

```diff
@@ -96,12 +102,12 @@ String getFieldValue() {
    */
   Set<String> getOutputFields() {
     Set<String> fields = new HashSet<String>();
+    Map<String, String> outputFieldMappings = getFieldAliases();
     for (String field : Splitter.on(',').trimResults().split(outputFields)) {
-      String[] value = field.split(":");
-      if (value.length == 1) {
-        fields.add(value[0]);
+      if (outputFieldMappings.containsKey(field)) {
```

Comment:
so outputFields contains the pre-aliased field name? This should be mentioned in the description, otherwise its not clear if the pre-aliased or the aliased name should be used.